app/views/layouts/application.html.erb

```diff
@@ -20,6 +19,14 @@
     <% end %>
     <%= render partial: './partials/flash_message' %>
     <%= yield %>
-    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+    <script>
+      $(document).ready(function() {
+        setTimeout(function() {
+          $("#notice-wrapper").fadeOut("slow", function() {
+            $(this).remove();
+          })
+        }, 4500);
+      });
+    </script>
```

Comment:
This function is the fade out javascript, notice-wrapper is applied to all flash messages so all messages will fade out which is awesome